content/teknologi/altinnstudio/altinn-api/_index.md

```diff
@@ -7,1012 +7,61 @@ aliases:
 
-```json {linenos=false,hl_lines=[2]}
-{
-    "id": "347829/762011d1-d341-4c0a-8641-d8a104e83d30",
-    "selfLinks": {
-        "apps": "{appPath}/instances/347829/41e57962-dfb7-4502-a4dd-8da28b0885fc",
-        "platform": "{platformPath}/instances/347829/41e57962-dfb7-4502-a4dd-8da28b0885fc"
-    },
-    "appId": "org/app",
-    "instanceOwner": {
-        "partyId": "347829",
-        "personNumber": "12247918309"
-    },
-    "created": "2019-03-06T13:46:48.6882148Z",
-    "createdBy": "org23",
-    "dueBefore": "2019-06-01T12:00:00Z",
-    "visibleAfter": "2019-05-20T00:00:00Z",
-    "title": { "nb": "Arbeidsmelding for Ola Nordmann"},
-    "process": {
-        "started": "2019-09-25T09:32:44.20Z",
-        "currentTask": {
-            "started": "2019-10-10T32:22.00Z",
-            "elementId": "Task_1",
-            "name": "Fyll ut",
-            "altinnTaskType": "data",
-            "validated": {
-                "timestamp": "2019-10-04T12:00.00Z",
-                "canCompleteTask": true
-            }
-        }
-    },
-    "status": {
-        "archived": null,
-        "softDeleted": null,
-        "hardDeleted": null
-    },
-    "completeConfirmations": [
-        {
-            "stakeholderId": "org",
-            "confirmedOn": "2020-03-16T14:38:11"
-        }
-    ],
-    "data": [
-    {
-        "id": "692ee7df-82a9-4bba-b2f2-c8c4dac69aff",
-        "dataType": "model1",
-        "contentType": "application/xml",
-        "blobStoragePath": "org/app/762011d1-d341-4c0a-8641-d8a104e83d30/data/692ee7df-82a9-4bba-b2f2-c8c4dac69aff",
-        "selfLinks": {
-            "apps":   "{appPath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/data/692ee7df-82a9-4bba-b2f2-c8c4dac69aff",
-            "platform": "{storagePath}/instances/347829/762011d1-d341-4c0a-8641-d8a104e83d30/data/692ee7df-82a9-4bba-b2f2-c8c4dac69aff"
-        },
-        "filename": "prefill.xml",
-        "created": "2019-03-06T15:00:23Z",
-        "createdBy": "org23",
-        "size": 20001,
-        "locked": false,
-    },
-    ...
-    ]
-}
-```
-
-## Data
-
-An instance holds metadata objects that describe the data files that can be uploaded and downloaded from storage. These metadata objects are called data elments.
-A data element is identified with a GUID. 
-
-- to stream a data file you must talks to the `/data` endpoint of the instance.
-- to get or update the data element metadata elements you must talk to the `/dataelements` endpoint of the instance.
-
-### Create and upload data
+Metadata for an App is the second job of the App API. There are endpoint that gives access to the metadata of the App itself, its data types and process deescription.
```

Comment:
```suggestion
Metadata for an app is the second job of the App API. There are endpoint that gives access to the metadata of the App itself, its data types and process deescription.
```

Reply:
or are you consciously refering to the app with a capital A?

Reply:
I have, but that doesn't make it right. The goal was to emphasis that it isn't just any app, but Altinn App. Not sure if it works.

Reply:
> that it isn't just any app, but Altinn App.

@SandGrainOne What do you mean by "Altinn App"? Having to ask, it probably does not work :-P

Reply:
@altinnadmin does you comment mean that you want us to write **app** and not **App**?

Reply:
When describing an app, or functionality or APIs related to an app, it should always be lowercase.
Uppercase only when we're referring to "Altinn Apps" (the infrastucture) or when App is used in namespace/packages.

You also seem to be using "App API" as a short-hand form for [Altinn.App.Api](https://www.nuget.org/packages/Altinn.App.Api/) ? Might cause confusion since "app API" will be the entire API surface of an app, which might be larger than Altinn.App.Api.